modules/pyspark/added/s2i/assemble

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+shopt -s dotglob
+
+# first run the original assemble script
+/usr/libexec/s2i/assemble.orig
+
+echo "---> Proceeding with customized assemble ..."
+
+# If the root of the repo contains file (worker-)requirements.txt, install them and bundle
+# them together into big zip file that will be passed to spark-submit.
+
+# note: the idea of worker-requirements.txt is to isolate those dependencies that
+# are necessary only for spark workers, because not all the dependencies driver needs
+# are also needed on the worker nodes
+r_file=`[ -f "$APP_ROOT/src/worker-requirements.txt" ] && echo worker-requirements.txt || echo requirements.txt`
```

Comment:
some discussion on slack ...

Let's change this so that only worker-requirements.txt get bundled up and distributed (requirements.txt will still be handled by the upstream assemble and will be installed locally, available to the driver)

This will let a user differentiate between driver deps and worker deps.  There might be cases where you have to repeat a dependency in both files to get it on the driver and the workers as well, but I think that's okay.  Thoughts?